### DISCUSSION TITLE
What columns by default in lfs 2 ?

### DISCUSSION THREAD

**Discussion by Canop** [Canop/dysk#29] [General]
As reference, the current version of lfs looks like this:

![image](https://user-images.githubusercontent.com/617006/150633386-401e85d4-f37c-479e-a3da-d0eba952a24e.png)

With the `label` and `inodes` columns:

![image](https://user-images.githubusercontent.com/617006/150642758-7aecdfb0-600f-498e-8e10-bc746191935d.png)

The next version of lfs should come with
* a launch argument syntax to specify exactly what columns to display
* less columns shown by default, because most users don't, for example, even read the `id` or `dev`

The syntax for choosing,adding,removing columns will be similar to the one of [rhit fields](https://dystroy.org/rhit/usage-fields/).

The big question that remains is **what fields to display by default ?**

Can you please give your input based on your typical and rare usages ?

**Comment by simonbcn** [comment]
`filesystem` | `used` | `use%` | `free` | `mount point`

**Comment by Canop** [comment]
Here's what it looks like:
![image](https://user-images.githubusercontent.com/617006/154805809-5745c3e2-cf0f-4e79-a9b6-6063138e66f9.png)

**Comment by Canop** [comment]
It's in the `cols` branch. I'll make a documentation website before I launch lfs 2 or I would have too many users lost facing the `--cols` argument.

**Comment by Canop** [comment]
lfs 2 is now released, and there's a new site : https://dystroy.org/lfs/

**Reply by danie-dejager** [reply]
The table page has not been added yet
https://dystroy.org/table

**Reply by Canop** [reply]
That was a wrong link. Fixed it.
